code/modules/pda/chatroom.dm

```diff
@@ -7,7 +7,7 @@ var/list/chatrooms = list(new /datum/chatroom("General Discussion"))
 	var/list/logs = list() // chat logs
 	var/topic = "Discussion" // topic message for the chatroom
 	var/is_public = 1
-	var/announcer = "CyberiadAI"
+	var/announcer = "Announcer"
```

Comment:
This has absolutely *no* fucking place in this PR.

Reply:
Once again, we will have other stations.

Reply:
It's out of scope of the pr. You also assume other stations before we have them. Plus even of it's another station they could be using software created on another station in this case the cyberiad